med-inventory/src/layout/NavBar.jsx

```diff
@@ -147,6 +183,7 @@ export default function NavBar(props) {
         open={open}
       >
         <div className={classes.toolbar}>
+        <img className={classes.logoImageOpen} src= "/images/siddha.jpg" alt="logo" />
```

Comment:
"logoImageOpen" doesn't give any meaning, please give meaningful names

Reply:
I thought the style represents logo when the drawer is open

Reply:
Maybe give "open-drawer-logo".Or something is meaningful